internal/pkg/service/cli/dialog/create_table.go

```diff
@@ -56,5 +58,57 @@ func (p *Dialogs) AskCreateTable(args []string, branchKey keboola.BranchKey, all
 		opts.PrimaryKey = primaryKey
 	}
 
+	filePath := p.options.GetString("columns-from")
+	if p.options.IsSet("columns-from") && len(opts.PrimaryKey) == 0 && opts.Name == "" {
+		createTableRequest, err := parseJSONInputForCreateTable(filePath)
```

Comment:
So if I specify primary key flag, then the `columns-from` file will be ignored?

Reply:
Related to: https://github.com/keboola/keboola-as-code/pull/1525#discussion_r1463042370